nh_eobs_mental_health/models/nh_clinical_patient_observation_ews.py

```diff
@@ -109,6 +113,12 @@ def schedule_clinical_review_notification(self, activity_id):
         activity_model = self.env['nh.activity']
         activity = activity_model.browse(activity_id)
         still_valid = self.is_refusal_in_effect(activity_id, mode='child')
+
```

Comment:
We don't have one at the moment. Can log at debug, not sure what level the servers run at though should say in `/etc/odoo/server.cfg`